src/doc/trpl/closures.md

```diff
@@ -181,6 +174,36 @@ fn main() {
 
 Doing this is not particularly common, but it's useful every once in a while.
 
+Before we move on, let us look at a function that accepts two closures.
+
+```{rust}
+fn compose<F, G>(x: i32, f: F, g: G) -> i32
+    where F: Fn(i32) -> i32, G: Fn(i32) -> i32 {
```

Comment:
When wrapping a `where` like this I usually see the `{` on the next line, but there's no official guideline.